lib/app.js

```diff
@@ -162,6 +167,21 @@ page('/search/:search', render('#content', require('./views/search.js')));
 
 page('/profile/', render('#content', require('./views/profile.js')));
 
+page('/validate/', function() {
+  var params = urlParams();
+  app.bugzilla.validLogin({
+    login: params.client_api_login,
+    token: params.client_api_key
+  }).then(function() {
+    app.user = {name: params.client_api_login};
+    localStorage.user = JSON.stringify({
+      login: params.client_api_login,
+      token: params.client_api_key
+    });
+    page('/');
```

Comment:
yup would prefer that too, I think it would need work on the bmo side though so will file as a follow up